### PR TITLE
Update teaser formats

### PR DESCRIPTION
**Pull request by thewilkybarkid**
https://github.com/elifesciences/issues/issues/5416

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Open for review?

**Comment by thewilkybarkid** [comment]
Always; in draft here as not mergable.

File: tests/src/ViewModel/TeaserTest.php [hunk 1]
```diff
@@ -50,8 +50,7 @@ public function it_can_load_main()
             null,
             TeaserFooter::forArticle(
                 $this->metaFromData($data['footer']['meta']),
-                $data['footer']['formats']['html'],
-                $data['footer']['formats']['pdf']
+                $data['footer']['formats']['list']
```
Comment (giorgiosironi) [inline tests/src/ViewModel/TeaserTest.php]: got that this indirectly comes from the `pattern-library`'s JSON files, good reuse

Comment (giorgiosironi) [reply tests/src/ViewModel/TeaserTest.php]: Not really, from the JSON files in this project. Still good and more customizable probably.

**Pull request merged**